packages/schema/src/utils/strip-quotes.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Strip leading/trailing quotes from a string
+ */
+export function stripQuotes(value?: string | null): string | null {
```

Comment:
```suggestion
export function stripQuotes(value: string | null): string | null {
```

Shouldn't allow undefined since trim will throw if used that way. Undefined was needed in `schema-inspector` but not here.

Reply:
Good catch